src/LocationData.cpp

```diff
@@ -15,52 +15,74 @@ namespace weatherserver {
 
     LocationData::LocationData(std::string name, std::string city, std::string countryCode, double latitude, double longitude,
         bool hasBeenReceivedWeatherData, bool isInitialized, bool isAddingWeatherToAddressSpace)
-        : name{ name }, city{ city }, countryCode{ countryCode }, latitude{ latitude }, longitude{ longitude },
-        hasBeenReceivedWeatherData{ hasBeenReceivedWeatherData }, isInitialized{ isInitialized }, isAddingWeatherToAddressSpace{ isAddingWeatherToAddressSpace }
+        : name { name },
+          city { city },
+          countryCode { countryCode },
+          latitude { latitude },
+          longitude { longitude },
+          hasBeenReceivedWeatherData { hasBeenReceivedWeatherData },
+          isInitialized { isInitialized },
+          isAddingWeatherToAddressSpace { isAddingWeatherToAddressSpace }
     {
         readLastTime = std::chrono::system_clock::now();
     }
 
-    LocationData::LocationData(std::string name, std::string countryCode)
-        : LocationData{ name, "", countryCode, INVALID_LATITUDE, INVALID_LONGITUDE }
-    {}
+	LocationData::LocationData()
+    : isInitialized(false),
```

Comment:
Inconsistent indentation for this member initializer list, compare with previous constructor.